Library/Formula/domoticz.rb

```diff
@@ -0,0 +1,58 @@
+class DomoticzDownloadStrategy < SubversionDownloadStrategy
+  def stage
+    cp_r "#{cached_location}/.", Dir.pwd
+  end
+end
+
+class Domoticz < Formula
+  homepage "http://www.domoticz.com/"
+  head "http://svn.code.sf.net/p/domoticz/code/trunk/", :using => DomoticzDownloadStrategy
+
+  depends_on "cmake" => :build
+  depends_on "boost" => :build
+  depends_on "openssl"
+  depends_on "libusb"
+  depends_on "libusb-compat"
+  depends_on "homebrew/dupes/zlib"
+
+  def install
+    system "sed", "-i", "-e", "s:DESTINATION\\ /opt/domoticz:DESTINATION\\ \\$\\{CMAKE_INSTALL_PREFIX\\}/opt/domoticz:", "CMakeLists.txt"
```

Comment:
Use inreplace. Also, can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!